### PR TITLE
EL tests refactor with jsp,servlet vehicles for Platform TCK 11

### PR DESCRIPTION
**Pull request by alwin-joseph**
**Fixes Issue**
https://github.com/jakartaee/platform-tck/issues/1380

**Describe the change**
EL Tests are refactored to run in JSP, Servlet vehicles for EE11 Platform TCK.
Total Tests run : 717

Test failures(when run with Glassfish 8.0.0-JDK17-M5) :

- 2 test failures in optionalelresolver (seems related to https://github.com/jakartaee/expression-language/pull/240 , should be resolved in EL 6.0.1 ) - tests passed with 8.0.0-JDK17-M7.

 - 16 tests failures are due to missing property that need to be added to arquillian-protocol-common jar : PR https://github.com/eclipse-ee4j/jakartaee-tck-tools/pull/147 for adding property.

### CODE REVIEW

File: el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/arrayelresolver/ELClientJspTest.java [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+  private static final String[] names = { "doug", "nick", "roger", "ryan",
+      "ed" };
+
+  public static void main(String[] args) {
+    ELClientJspTest theTests = new ELClientJspTest();
+    Status s = theTests.run(args, System.out, System.err);
+    s.exit();
+  }
+
+  public void setup(String[] args, Properties p) throws Exception {
```
Comment (alwin-joseph) [inline el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/arrayelresolver/ELClientJspTest.java:145]: @starksm64 I see that the setup method is not invoked to set the properties for all the ClientTest classes. Do you think I am missing anything here?
cc @gurunrao

Comment (gurunrao) [reply el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/arrayelresolver/ELClientJspTest.java:145]: Please verify if properties for EL tests are present @ https://github.com/eclipse-ee4j/jakartaee-tck-tools/blob/bf59506a644b4194ce4528e6defe66465b46a8a3/arquillian/common/src/main/java/tck/arquillian/protocol/common/TsTestPropsBuilder.java#L21 
Javatest protocol will only read properties present at https://github.com/eclipse-ee4j/jakartaee-tck-tools/blob/bf59506a644b4194ce4528e6defe66465b46a8a3/arquillian/common/src/main/java/tck/arquillian/protocol/common/TsTestPropsBuilder.java#L21

Comment (alwin-joseph) [reply el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/arrayelresolver/ELClientJspTest.java:145]: Okey. I see "variable.mapper" is missing which is required for EL tests. Can it be added.

Comment (gurunrao) [reply el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/arrayelresolver/ELClientJspTest.java:145]: @starksm64 - Please release new jakartaeetck-tools with above property.

Comment (alwin-joseph) [reply el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/arrayelresolver/ELClientJspTest.java:145]: https://github.com/eclipse-ee4j/jakartaee-tck-tools/pull/147

**Review by starksm64 — APPROVED** [review]

**Pull request merged**